tests/js/spec/views/eventsV2/table/tableView.spec.jsx

```diff
@@ -97,6 +107,30 @@ describe('TableView > CellActions', function () {
     ProjectsStore.reset();
   });
 
+  it('updates sort order on equation fields', async function () {
```

Comment:
These could definitely be better tests. I wanted to check browserHistory.replace gets called correctly and the resulting eventView has the right sort but I wasn't able to get it working. Pushing this out for now to get the fix in.